fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1683,9 +1684,13 @@ private void buildPartitions(Database db, OlapTable table, List<Partition> parti
             numReplicas += partition.getReplicaCount();
         }
 
-        if (partitions.size() >= 3 && numAliveBackends >= 3 && numReplicas >= numAliveBackends * 500) {
+        if (table.isLakeTable() ||
```

Comment:
why add `isLakeTable` check?

Reply:
because lakeTable can not decide whether to call buildPartitionsConcurrently or buildPartitionsSequentially according to numAliveBackends. So we just choose to use buildPartitionsConcurrently if it is a lake table